src/Library/Calificacion.cs

```diff
@@ -9,8 +9,17 @@ public class Calificacion
     public Cliente Calificado {get; set;}
     public Calificacion(int value, Cliente calificador, Cliente calificado)
     {
-        this.Value = value;
-        this.Calificador = calificador;
-        this.Calificado = calificado;
+        if (value<1 || value>5)
```

Comment:
don't check the negative:
`(value >= 1 || value <= 5)`

Put the throw new Exception within the `else`